css/style.css

```diff
@@ -0,0 +1,84 @@
+/**
+ * Place your custom styles here.
+ */
+
+ /*GLOBAL OVERRIDES*/
+@import url('https://fonts.googleapis.com/css?family=Raleway');
+@import url('https://fonts.googleapis.com/css?family=Oswald');
```

Comment:
I was alternating between News Cycle and Oswald for headings. I will be switching between the two for testing purposes. It's likely News Cycle will win though.